test/project_types/script/layers/application/extension_points_test.rb

```diff
@@ -106,7 +106,7 @@
       let(:scripts_beta_extension_points) { true }
 
       it "returns a list of all languages implemented by all extension points" do
-        assert_equal ["assemblyscript", "rust", "tinygo"], subject
+        assert_equal ["assemblyscript", "rust", "wasm", "tinygo"], subject
```

Comment:
With the current languages we support, the mis-ordering here wouldn't matter much. But if we wanted to be complete, in the method we could simply sort `wasm` to the back like `["assemblyscript", "rust", "wasm", "tinygo"].partition { |v| v != "wasm" }.flatten`

Reply:
I think I'll leave it as is. Keeping like this is fine, and treating wasm like any other language when we can (as opposed to writing special code for it) makes sense to me.